vagrant-centos-basic/Vagrantfile

```diff
@@ -0,0 +1,17 @@
+Vagrant.configure("2") do |config|
+    config.vm.define "centos-basic" do |subconfig|
+        subconfig.vm.box = "centos/7"
+        subconfig.vm.hostname = "centos-basic"        
+        subconfig.vm.network "private_network", type:"dhcp"
+        subconfig.vm.synced_folder ".", "/vagrant", disabled: true        
+        subconfig.vm.provider "hyperv" do |h|            
+            h.differencing_disk = true
+            h.cpus = 4
+            h.memory = 2048
+            h.maxmemory = 4096
+            h.vmname = "centos-basic"
+            h.vlan_id = 7
```

Comment:
`vlan_id = 7` is referenced throughout. Should that be parameterized?

Reply:
Yes. I'll make a variable